app/experimenter/nimbus-ui/src/components/PageHome/index.tsx

```diff
@@ -40,6 +43,13 @@ export const Body = () => {
   }
 
   if (error) {
+    if (!hasRefetched) {
+      setTimeout(() => {
+        refetch();
+        setHasRefetched(true);
+      }, REFETCH_DELAY);
+      return <RefetchAlert />;
```

Comment:
So we use this logic for fetching experiments but not configuration on the home page?  What's the reasoning there?

Reply:
Ohh, good call out. I actually just forgot that I need to add this to `App/index.tsx` as well to cover both queries, will add it there.